waspc/examples/todoApp/todoApp.wasp

```diff
@@ -8,7 +8,14 @@ app todoApp {
   ],
   auth: {
     userEntity: User,
-    methods: [ UsernameAndPassword ],
+    externalAuthAssociationEntity: SocialLogin,
```

Comment:
Not sold on this name, `externalAuthAssociationEntity`, but it is pretty accurate at least. 🤔